falcon_marshmallow/middleware.py

```diff
@@ -264,24 +285,32 @@ def _get_schema(cls, resource, method, msg_type):
             return specific_schema
         return getattr(resource, "schema", None)  # type: ignore
 
-    @classmethod
-    def _content_type_is_json(cls, content_type):
+    def _content_is_expected_type(self, content_type):
         # type: (str) -> bool
-        """Check if the provided content type is json. This uses similar code to client_accepts in falcon.request.
-        If content type is not provided, assume json for backwards compatibility.
+        """Check if the provided content type is json.
```

Comment:
@mplanchard this docstring still has some specific references to "json" (here on line 290, and also on line 294)